test/nn/test_load_state_dict.py

```diff
@@ -362,7 +386,123 @@ def test_load_state_dict_warn_assign(self):
             m.load_state_dict(state_dict)
 
 
+def load_torch_function_handler(cls, func, types, args=(), kwargs=None):
+    kwargs = {} if kwargs is None else kwargs
+
+    def module_load(dest, src):
+        # always convert src to cls
+        if isinstance(dest, cls):
+            if type(src) is torch.Tensor:
+                return cls(src)
+            elif type(src) is cls:
+                return src
+            else:
+                if isinstance(src, MyWrapperLoadTensor):
+                    return cls(src._data)
+                return cls(src)
+        else:
+            return src
+
+    if func is torch.Tensor.module_load:
+        return module_load(*args, **kwargs)
+    else:
+        with torch._C.DisableTorchFunctionSubclass():
+            # detach must return instance of same subclass for nn.Parameter()
```

Comment:
Is MyWrapperLoadTensor actually using this function though?

Reply:
My understanding is when creating `Parameter(MyWrapperLoadTensor)`

`MyWrapperLoadTensor.detach()` would call into `MyLoadTensor.__torch_function__` which would then go into the `__torch_dispatch__` for `MyWrapperLoadTensor.detach()`, so I do think it will go into this, yes.

Reply:
Ok!